### DISCUSSION TITLE
changelog/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#14] [General]
# changelog/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/changelog/

**Comment by HobbitQia** [comment]
我超，什么神！

**Comment by smd1121** [comment]
这个好！准备偷了！

**Comment by stormckey** [comment]
太强啦

**Comment by QAQdev** [comment]
hello, 我在我自己的mkdocs站点里使用了自定义字体，大概ttf格式在40MB，woff格式在20MB，加载较慢。不知道有没有什么方法可以加速加载🤔我没有云服务器，因此cdn加速就没尝试过，另外尝试过的方法有preload标签和压缩字体文件（

**Reply by TonyCrane** [reply]
不是很懂为什么要在这个页面下评论问问题，为什么不去 issue 呢

我去你网站上看了一下，貌似只有首页那一句诗，如果是固定的话那试试 [allanguys/font-spider-plus](https://github.com/allanguys/font-spider-plus) 按需压缩字体就好。但看起来好像不固定，那简单点的话建议你换字体，非要用这个的话可以考虑把字体文件拆成多份文件配合 css 实现按需、并行加载，比如你看这个 css：<https://cdn.tonycrane.cc/lxgw/lxgwscreen.css>（但我没实操过，搜了搜或许可以试试 [Lambholl/webfont-seperator](https://github.com/Lambholl/webfont-seperator/)）

**Reply by QAQdev** [reply]
sry，忘记 issue 了。
我再去研究下你提供的方案，感谢！

**Comment by TajangSec** [comment]
好精致啊，卡片还有各种功能。我去年年底自己写了一个hugo的主题，UI和配色仿的HTB，虽然是自己写的，但是很多前端知识还是没学懂，都是社区找的示例。跟你这个比更是远远不如。我的博客：[h4cker.zip](https://h4cker.zip)
